CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Bolt Next
 
+### Breaking Changes
```

Comment:
Note: I haven't seen a "Breaking changes" header before, I'm checking that it's standard? Will we continue to use it as needed?

Reply:
hmm I think deprecations may be the standard header